onnxruntime/core/mlas/lib/convolve.cpp

```diff
@@ -744,7 +744,8 @@ MlasConvTryMultithread(
     const float* Filter,
     const float* Bias,
     float* WorkingBuffer,
-    float* Output
+    float* Output,
+    const ThreadPool* ExternalThreadPool
```

Comment:
ExternalThreadPool is not used on Linux. It will generate a warning.

Reply:
Fixed.